docs/zh/guides/improving-query-performance/sparse-primary-indexes.md

```diff
@@ -370,6 +370,25 @@ UserID.bin，URL.bin，和EventTime.bin是<font face = "monospace">UserID</font>
 - 索引条目(索引标记)不是基于表中的特定行，而是基于颗粒。例如，对于上图中的索引条目‘mark 0’，在我们的表中没有<font face = "monospace">UserID</font>为240.923且<font face = "monospace">URL</font>为“goal://metry=10000467796a411…”的行，相反，对于该表，有一个颗粒0，在该颗粒中，最小<font face = "monospace">UserID</font>值是240.923，最小<font face = "monospace">URL</font>值是“goal://metry=10000467796a411…”，这两个值来自不同的行。
 
 - 主索引文件完全加载到主内存中。如果文件大于可用的空闲内存空间，则ClickHouse将发生错误。
+
+- 为什么图中显示的数据与结论不符合？因为图中的数据是禁用了自适应索引粒度后得到的，默认情况下索引粒度是自适应的。
+- 下面是英文官方文档和版本更新的描述信息。
+- https://clickhouse.com/docs/en/optimize/sparse-primary-indexes
+- We mentioned in the beginning of this guide in the "DDL Statement Details", that we disabled adaptive index granularity (in order to simplify the discussions in this guide, as well as make the diagrams and results reproducible).
+- For tables with adaptive index granularity (index granularity is adaptive by default) the size of some granules can be less than 8192 rows depending on the row data sizes.
+- 我们在本指南开头的“DDL 语句详细信息”中提到，我们禁用了自适应索引粒度（为了简化本指南中的讨论，并使图表和结果可重现）。
```

Comment:
seems you should put this to the above note (line 340)

Reply:
Yes,i think it's a good idea,this has already been mentioned above.Maybe I should add a new note behind line 340 for readers to understand better?

Reply:
OK,I'll remove the quoat English text and think carefully about the translation.